plugin-api/src/main/java/io/everytrade/server/plugin/api/parser/BuySellImportedTransactionBean.java

```diff
@@ -29,7 +29,7 @@ public BuySellImportedTransactionBean(
         );
 
         Objects.requireNonNull(this.baseQuantity = baseQuantity);
-        Objects.requireNonNull(this.unitPrice = unitPrice);
+        this.unitPrice = unitPrice;
```

Comment:
Please add comment note into the code that if unit price is not specified (null) it will be automatically taken from the market.